generator settings/VMLbackground.md

```diff
@@ -0,0 +1,26 @@
+
+VML is stands for Vector Markup language and this format is used for define shapes that renders correctly on Word engined Outlook versions. From Outlook2007 to the newest versions, Outlooks only render the background images or the rounded shape buttons if they were declared in VML.
+
+Inside the settings object you can now place a vmlBackground parameter either of the values true or false:
+
+-If it set to "true", the generator will generate declarations for the background in HTML and VML as well. VML code will be placed inside coditional comments to ensure that only Outlook email clients notice that.
```

Comment:
I would use `true` because that will highlight the text within as code. "true" would mean that it's a string, but it's not, it's a boolean.